miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebViewClient.kt

```diff
@@ -33,20 +37,31 @@ internal class MiniAppWebViewClient(
         request: WebResourceRequest,
         error: WebResourceError
     ) {
-        if (request.url != null && request.url.toString().startsWith(customScheme)) {
-            loadWithCustomDomain(view, request.url.toString().replace(customScheme, customDomain))
-            return
+        if (request.url != null) {
+            val requestUrl = request.url.toString()
+            if (requestUrl.startsWith(customScheme))
+                return onErrorRedirect(view) {
+                    loadWithCustomDomain(view, requestUrl.replace(customScheme, customDomain))
+                }
+            else if (requestUrl.startsWith("tel:"))
```

Comment:
`shouldOverrideUrlLoading()` does not work in our sdk. I don't know why, maybe because of using WebAssetLoader.
There is advantage of handling this in `onReceivedError()`, we can stop the redirection of showing error page on webview.